pylti/flask.py

```diff
@@ -152,7 +154,7 @@ def lis_result_sourcedid(self):
         return session['lis_result_sourcedid']
 
     @property
-    def role(self):
+    def role(self):  # pylint: disable=no-self-use
```

Comment:
@symbolist this is @property. these are object methods no need to convert into static